extensions/wrapper/wrapper/src/test/java/de/sovity/edc/ext/wrapper/api/ui/pages/dashboard/DashboardPageApiServiceTest.java

```diff
@@ -65,16 +66,16 @@ class DashboardPageApiServiceTest {
         testDatabase -> {
             config = forTestDatabase("my-edc-participant-id", testDatabase);
 
-            config.setProperty("edc.oauth.token.url", "https://token-url.daps");
-            config.setProperty("edc.oauth.provider.jwks.url", "https://jwks-url.daps");
+            config.setProperty(ConfigProps.EDC_OAUTH_TOKEN_URL, "https://token-url.daps");
```

Comment:
There's a lot of `ConfigProps.`

Using `import de.sovity.edc.utils.config.ConfigProps.*;` would be fine as the name of the prop is already exactly its env var name, so we know exactly what it represents.

Reply:
Nah, I would not start with asterisk Imports